spec/models/reporting/patient_visits_per_month_spec.rb

```diff
@@ -0,0 +1,81 @@
+require "rails_helper"
+
+RSpec.describe Reporting::PatientVisitsPerMonth, {type: :model, reporting_spec: true} do
+  describe "Associations" do
+    it { should belong_to(:patient) }
+  end
+
+  describe "the visit definition" do
+    it "considers a BP measurement as a visit" do
+      bp = create(:blood_pressure, :with_encounter, recorded_at: test_times[:now])
```

Comment:
I'd love to eventually make `:with_encounter` just the default

Reply:
I think we could do this in a different PR. Added a [story](https://app.clubhouse.io/simpledotorg/story/3899/change-bp-and-bs-factories-to-create-encounters-by-default) for it.